marge/job.py

```diff
@@ -217,7 +217,7 @@ def wait_for_ci_to_pass(self, source_project_id, commit_sha):
             if ci_status not in ('pending', 'running'):
                 log.warning('Suspicious build status: %r', ci_status)
 
-            log.info('Waiting for %s secs before polling CI status again', waiting_time_in_secs)
+            log.debug('Waiting for %s secs before polling CI status again', waiting_time_in_secs)
```

Comment:
@benjamb This is a good idea, but we should probably add something like `log.info("Waiting for CI to pass...)` before the while loop, so we can check the logs and easily tell what marge is doing.

Reply:
@jcpetruzza You're right. I'll update shortly.

Reply:
Done!

Reply:
Excellent, thanks!